website/snippets/_git-providers-supporting-ci.md

```diff
@@ -8,8 +8,8 @@ The following table outlines the available integration options and their corresp
 
 | **Git provider** | **Native dbt Cloud integration** | **Automated CI job**|**Git clone**| **Information**|
 | -----------------| ---------------------------------| -------------------------------------------|-----------------------|---------|
-|[Azure DevOps](/docs/cloud/git/connect-azure-devops)<br /> <Lifecycle status="enterprise" />|  ✅|  ✅ |  ✅  | Organizations on the Team  and Developer plans can connect to Azure DevOps using a deploy key. Note, you won’t be able to configure automated CI jobs but you can still develop.|
-|[GitHub](/docs/cloud/git/connect-github)<br /> <Lifecycle status="developer,team,enterprise" /> | ✅ | ✅ |  ✅         |                                  
-|[GitLab](/docs/cloud/git/connect-gitlab)<br /> <Lifecycle status="developer,team,enterprise" /> | ✅ | ✅ |  ✅         |
+|[Azure DevOps](/docs/cloud/git/connect-azure-devops)<br /> <Lifecycle status="managed" />|  ✅|  ✅ |  ✅  | Organizations on the Team  and Developer plans can connect to Azure DevOps using a deploy key. Note, you won’t be able to configure automated CI jobs but you can still develop.|
+|[GitHub](/docs/cloud/git/connect-github)<br /> <Lifecycle status="developer,self_service,managed" /> | ✅ | ✅ |  ✅         |                                  
```

Comment:
Note for later really, these don't look so good in the table format. Suggest we avoid using the pills in this fashion and instead have a separate column that lists supported tiers if needed.

Reply:
i agree and prefer having a separate column as well! i can create a follow up pr to change this